pom.xml

```diff
@@ -0,0 +1,106 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>eclipse.asciidoctor.editor</groupId>
+  <artifactId>releng</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <pluginRepositories>
+    <pluginRepository>
+      <id>tycho-snapshots</id>
+      <url>https://repo.eclipse.org/content/repositories/tycho-snapshots/</url>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <properties>
+    <tycho.version>2.5.0-SNAPSHOT</tycho.version>
```

Comment:
Ahem - is this another version (the plugin version) - I am a little bit confused. My comment before was about the versions, but maybe this was only the POM version and this one the tycho plugin version ...

Hmm.. to keep things simple I would normally prefer to keep both versions in sync, so it would be always clear which pom is for which eclipse plugin version...

What's your opinion?

Reply:
This is the Tycho version used for the build, so we should not have that in sync with the Gradle editor version